scalyr_agent/builtin_monitors/linux_process_metrics.py

```diff
@@ -846,13 +846,13 @@ def _initialize(self):
         self.__target_pids = []
```

Comment:
Change `__target_pids` to some other variable name.  It's confusing to have `__target_pids` and `self.__target_pids`.